### DISCUSSION TITLE
Modal not showing when lazy loading

### DISCUSSION THREAD

**Discussion by webmclink** [livewire/livewire#8546] [1. Help]
The modal is not showing when I set the component to lazy loading. below is my code

Route::get('/test', Test::class)->name('test.index')->lazy();

but when I remove the ->lazy(); the modal is showing. How to make it able to show with lazy loading? Thanks

**Comment by lrljoe** [comment]
You'll need to share a bit more than just that! 

 What kind of modal are you opening, what event(s) trigger it to open etc
